src/services/ApiService/MembershipApi.ts

```diff
@@ -16,15 +16,15 @@ export const formatResponse = (timestamps: PromiseType<ReturnType<typeof getUnlo
   // Convert to Bigies so we can determine their expiry date.
   // @todo: prefer date-fns for time comparaisons.
   const expiries = pipe(
-    mapObjIndexed(mapObjIndexed(toString)),
-    mapObjIndexed(mapObjIndexed(bigify))
+    mapObjIndexed(mapObjIndexed(bigify)),
```

Comment:
Can we keep these as BN's please? They are a result of a contract call and thus can theoretically be larger than what can fit in an int. (you parse these later as int)

Reply:
The contract returns a BN for a unix timestamp
It's a contract which we control so I don't see the risk of an underflow/overflow.
Within the app, redux encourages us to only store serializable values.
The value is a date and should be treated as such.
This is why it seems best to store the value as a string and use date operations.

Reply:
We don't control the contract the BN comes from. At the end of the day it is a returned result from an unlock contract.